### ISSUE TITLE
待翻-如何处理面试压力（How to Handle Stress During a Job Interview）

### ISSUE THREAD

**Issue by lietoumai** [lietoumai/Hunter#14]
Have you ever felt incredibly stressed during a job interview? If so, you’re not alone. Most people say that interviewing for a job is an overwhelmingly stressful experience. Fortunately, you can come to terms with job interview stress by understanding that a certain amount of stress can actually help you ace the interview; that practicing for your interview can help you maintain a manageable stress level; and that there are some effective responses you can use if your level of stress starts to feel overwhelming.

First, consider the upside of stress: Researchers and counselors remind us that an appropriate amount of stress can be a positive thing, while too much stress can wear us out mentally, emotionally, and physically. It can be a balancing act to keep your stress level at an appropriate, productive level. A job interview provides the perfect example of how just enough stress can keep us on our toes while helping us to put our best foot forward.

Lyrics from the Broadway smash hit A Chorus Line demonstrate how much is at stake in an audition — the stage version of an interview. The dancers sing, “I really need this job; I’ve got to get this job” as they execute difficult moves and literally remain on their toes. While they, and we, need to remember that there are other jobs out there, the stress of really wanting this particular job can help us approach the audition or interview with as much energy as possible. We just have to remind ourselves to use that stress effectively — remember, you want that job.

The problem is that job interviews are an unusual kind of conversation: one that we have only so often, where there’s a huge imbalance of power, and that requires the type of confident recitation of our strengths that wouldn’t fly in other social interactions. Unlike a relaxed chat with a friend, this encounter requires that you rattle off all your relevant skills, experiences, strengths, and interests in an efficient and effective manner so that your listeners are quickly reassured that their time with you is worthwhile (and might even be extended). Producing all of this information in a way that comes across as confident, eloquent, and appropriate means paying attention to your interviewer’s questions, striving to remember that list of things you wanted to be sure to say, and working hard to tackle unexpected questions. A bit of stress keeps you paying attention and giving your best.

Because a job interview isn’t typically something that we encounter very often, it’s important to fully prepare for this unusual and nonstandard interaction. Consider some of the standard things your interviewer is looking for: what do you already know how to do, how confident are you that you can learn new skills, what do you consider to be your strengths, and what might be a weakness that you’ve had to address. The latter is where you can demonstrate that you have self-awareness and know that no one is perfect but practice makes us better. Practice your responses to questions like these so that you know what you want to say when some version of them comes up. When you’ve already practiced these expected interactions, your stress level will be manageable and can even provide the remaining energy needed for the unexpected requests that come out of left field and require you to think on your feet. Clients of mine have said that practice gave them the confidence they needed and helped them to tackle the unusual surprise question.

But what happens when, despite all of your practice, you feel like you’ve screwed up during your interview? Your stress level might skyrocket and send you into a spiral of despair, making you feel that you’re flubbing the rest of the interview and that you’ve already lost the job. While you really can’t ask for a total do-over, rest assured that corrections can be made. When you realize that you forgot to say something important, you can still correct the omission in order to minimize ongoing stress. If you’re still in the interview, it’s always possible to say: “I just realized that I hadn’t mentioned…” Now you’ve made sure that the information you wanted to share has been shared. If you realize an important omission after the interview has already ended, you can send a thank-you email that says, “I want to add to, or clarify, or revise what I said about x…” Again, you’ve completed the message you wanted to get across. Now you don’t have to lie awake at night worried about that omission or mistake. A client of mine said that a hiring manager appreciated his willingness to admit his mistake in the interview and gave him the job because (1) he was qualified and (2) he hadn’t given up.

Sometimes, the stress level in some interviews goes way beyond what most people experience. As you can imagine — or perhaps have experienced — a room full of multiple interviewers can be difficult for the lone candidate. For example, a client of mine named Jane, who is very skilled in her field and very good at interacting with others, is an introvert who’s most comfortable in a one-on-one situation. She learned that she would have to appear before a panel of interviewers, and wanted to discuss what she could do in that situation to maintain a calm and confident demeanor. Jane’s concern was that she would not be able to “connect” effectively with every person in the room as she worked hard to answer their questions and tried to remember who was who.

We discussed making a quick outline in her notebook of the oval table where her interviewers sat, marking, as they introduced themselves, their positions around the table with their names and titles. Jane did just that, and her interviewers were pleased that she was able to address each one of them during the course of the interview. Jane also planned to make eye contact with each member of the group as they addressed her or heard her comments. By planning these ways of managing the stress of a multiperson interview team, she felt more confident and not overly stressed. Her interviewers, in turn, felt that she managed to engage with them all and took the extra effort to get to know them right away.

Another onslaught of overwhelming stress can occur when a wacky question comes your way. You didn’t see it coming (because who would?), you’re not sure if it’s sincere or intended as a joke, and you have to decide quickly how to attempt an answer that may be what your listener is seeking. Your first step in managing your stress and developing your response is to acknowledge the unusual nature of the question: “That’s an interesting question. May I have a moment to consider that?” This procedure could help by first finding out if the interviewer really means for you to answer the question. Is it just a joke to break the ice, or is it a sincere effort to find out if you can think on your feet? If the latter, their response gives you more time to think about a topic you hadn’t rehearsed.

When Ellen, a former client of mine, was asked what kind of plant she would be if indeed she were a plant, she replied with, “Hmm…that’s very interesting. Let me think about that for a moment.” Her listener responded with an enthusiastic nod and waited patiently. When Ellen responded after a pause, she said, “I enjoyed thinking about that; I always enjoy considering new ideas, and I think I would be a cactus. That kind of plant is sturdy: It plants itself firmly and it doesn’t require a lot of water. Rain or shine, cold or heat, it keeps its reliable place and can even supply some moist prickly pear to someone lost in the desert. Similarly, I work hard and understand the need to stay on my watch and to help others be successful, too.”

Wacky questions or not, an intense and important interview could create a stress level that results in brain freeze: Your mind goes blank, you stutter, or you blush with embarrassment. What to do?

Take a breath, take a sip of water, and take a moment to compose yourself. It’s okay to reply — even to a standard question — with, “Ah, let me think about that for a moment” or “Do you mean…” or “Could you rephrase the question?” You can also ask: “Does my answer cover the issues you were asking about?”

Regain some confidence and a sense of calm by remembering that you are interviewing the organization just as much as it is interviewing you. You are not bereft of all control. You do have some control over the conversation and some good questions to pose to your interviewers. After all, you want to know whether the job and the organization can really offer what you hope to find in your next job.

The best defense against experiencing overwhelming stress in a job interview is a good offense. Practice in advance what you want to be sure to say, whether it’s initially asked for or not. Remind yourself of your value, your skills, and your ability and enthusiasm about learning additional skills. If you tend to suffer from anxiety or lack of confidence in interviews or in general, consult with a career couch or a counselor who can help you prepare emotionally for this kind of situation. Remind yourself that you might not necessarily get the job, but you’ll know that you’ve done your best to stay calm and ace the interview if it’s at all possible.

If you tend to “sweat it out” literally or figuratively, make sure you’re dressed comfortably, in clothes and shoes that allow you to breathe easily and to focus on the subject at hand.

Remind yourself that stress in an interview is not only normal — it’s necessary. Prepare yourself for stress while you train yourself to do a great job. Take the edge off through practice beforehand and by successfully managing your stress in real time. Having nailed down what you want to say about your qualifications and having prepared for those tough questions, you’ll be ready to take that deep breath and know that you can manage an unexpected challenge. Use your stress effectively and land the job you want.

原文出处：https://hbr.org/2017/07/how-to-handle-stress-during-a-job-interview

**Comment by 15814** [comment]
上文中出现了重复段落。

>我们通过讨论并建议Jane在椭圆形桌子上对每个面试官做个标记。简这样做了，她的面试官很高兴她能在面试的过程中对每一个人讲话。当他们向她讲话或听到她的评论时，简还会尽力尝试与该组织的每一个成员进行眼神交流。通过这个方法来管理多人面试的压力，她感觉更自信，也不太紧张。而且她的面试官反过来觉得，她设法与他们所有人接触，并付出了额外的努力，并且想了解他们。

>我们通过讨论并建议Jane在椭圆形桌子上对每个面试官做个标记。简这样做了，她的面试官很高兴她能在面试的过程中对每一个人讲话。当他们向她讲话或听到她的评论时，简还会尽力尝试与该组织的每一个成员进行眼神交流。通过这个方法来管理多人面试的压力，她感觉更自信，也不太紧张。而且她的面试官反过来觉得，她设法与他们所有人接触，并付出了额外的努力，并且想了解他们。

**Comment by lietoumai** [comment]
> 上文中出现了重复段落。
> 
> > 我们通过讨论并建议Jane在椭圆形桌子上对每个面试官做个标记。简这样做了，她的面试官很高兴她能在面试的过程中对每一个人讲话。当他们向她讲话或听到她的评论时，简还会尽力尝试与该组织的每一个成员进行眼神交流。通过这个方法来管理多人面试的压力，她感觉更自信，也不太紧张。而且她的面试官反过来觉得，她设法与他们所有人接触，并付出了额外的努力，并且想了解他们。
> 
> > 我们通过讨论并建议Jane在椭圆形桌子上对每个面试官做个标记。简这样做了，她的面试官很高兴她能在面试的过程中对每一个人讲话。当他们向她讲话或听到她的评论时，简还会尽力尝试与该组织的每一个成员进行眼神交流。通过这个方法来管理多人面试的压力，她感觉更自信，也不太紧张。而且她的面试官反过来觉得，她设法与他们所有人接触，并付出了额外的努力，并且想了解他们。

感谢，已更正
